src/components/mirror.js

```diff
@@ -0,0 +1,67 @@
+import { Reflector } from "three/examples/jsm/objects/Reflector";
+
+/** In Reference, camera-tool.js line 605 and line 663 */
+function prepareSetPlayerHeadVisible() {
+  /** @return { THREE.Object3D? } */
+  const getPlayerHead = () => document.getElementById("avatar-head")?.object3D;
+
+  let playerHead = getPlayerHead();
+
+  /**
+   * @param { boolean } visible
+   */
+  return function(visible) {
+    if (!playerHead) {
+      playerHead = getPlayerHead();
+      return;
+    }
+
+    const scale = visible ? 1 : 0.0001;
+
+    playerHead.visible = visible;
+    playerHead.scale.set(scale, scale, scale);
+    playerHead.updateMatrices(true, true);
+    playerHead.updateMatrixWorld(true, true);
+  };
+}
+
+const setPlayerHeadVisible = prepareSetPlayerHeadVisible();
+
+/**
+ * Should need to entity that has geometry primitive
+ *
+ * @example
+ * ```html
+ * <a-plane mirror scale="5 5 5" position="0 3 0"></a-plane>
+ * <a-circle mirror="color: #7f7f7f" scale="5 5 5" position="0 3 0"></a-circle>
+ * ```
+ */
+AFRAME.registerComponent("mirror", {
+  schema: {
+    color: { type: "color", default: "#7f7f7f" }
+  },
+
+  init() {
+    const geometry = this.el.object3DMap?.mesh?.geometry;
+    if (!geometry) {
+      console.error('can not find "object3DMap.mesh.geometry"');
```

Comment:
Strings in the code are using double quotes. How about using single quotes consistently?

```suggestion
      console.error("can not find 'object3DMap.mesh.geometry'");
```

Reply:
<img width="412" alt="image" src="https://user-images.githubusercontent.com/41536271/157409724-2f026399-dacd-43de-a620-5e1d5b9198d4.png">

Project's eslint rule T.T

Reply:
Ah, I saw it wrong.
accept it. Thanks